### DISCUSSION TITLE
Are `repoGroups` ordered?

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#13087] [Request Help] [answered]
I'm looking into contributing another monorepo preset, for which I'd add an entry to the `repoGroups` map in `lib/config/presets/internal/monorepo.ts`. Now I'm looking at the suitable line to do so, but I can't seems to identify the order in which the keys or values are sorted by. Sublists appear to be sorted (looking at the first few characters at least), but then smaller and bigger jumps backwards and forwards in the alphabet are made.

Is there an order that I'm missing? If not, would you be open to a PR that does set AND enforce to maintain a clear and stable order?

**Comment by viceice** [comment, marked as answer]
it should be ordered, but nobody did it yet, feel free to open a pr 🤗
